documentation/book/con-certificates.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+= Certificates
+
+Each {ProductName} component needs its own private and public keys in order to support encryption.
+The public key has to be signed by a certificate authority (CA) in order to have a related X.509 certificate for providing server authentication and encrypting the communication channel with the client (which could be another broker as well).
+All component certificates are signed by a Certification Authority (CA) called _cluster CA_.
+Another CA is used for authentication of Kafka clients connecting to Kafka brokers.
+This CA is called _clients CA_.
+The CAs themselves use self-signed certificates.
+
+All of the generated certificates are saved as Secrets in the {ProductPlatformName} cluster, named as follows:
+
+`<cluster-name>-cluster-ca`::
```

Comment:
You should use _italic_ for the part which should be replaced (I don't care but docs will ;-)).